geosyspy/services/analytics_processor_service.py

```diff
@@ -0,0 +1,567 @@
+                raise ValueError()
+                break
+
+        return task_status
+
+    def launch_planted_area_processor(self,
+                                      start_date: str,
+                                      end_date: str,
+                                      seasonfield_id: str):
+        """launch a planted area analytics processor and get the job id in result
+
+            Args:
+                start_date (str) : the start date used for the request (format YYYY-MM-dd)
+                end_date (str) : the end date used for the request (format YYYY-MM-dd)
+                seasonfield_id (sfd) : seasonField geosys uniqueId
+
+            Returns:
+                taskId (str)
+        """
+
+        payload = {
+            "parametersProfileCode": "planted_area_default",
+            "data":
+                [
+                    {
+                        "id": seasonfield_id + "@ID",
+                        "start_date": start_date,
+                        "end_date": end_date
+                    }
+                ]
+        }
+
+        # format payload in json
+        # payload = json.dumps(payload)
+
+        processor_endpoint: str = urljoin(self.base_url,
+                                          GeosysApiEndpoints.LAUNCH_PROCESSOR_ENDPOINT.value.format("planted_area"))
+
+        response = self.http_client.post(processor_endpoint, payload)
+
+        if response.ok:
+            task_id = json.loads(response.content)["taskId"]
+            return task_id
+        else:
+            logging.info(response.status_code)
+            raise ValueError(response.content)
+
+    def launch_harvest_processor(self,
+                                 season_duration: int,
+                                 season_start_day: int,
+                                 season_start_month: int,
+                                 crop: str,
+                                 year: int,
+                                 seasonfield_id: str,
+                                 geometry: str,
+                                 harvest_type: Harvest):
+        """launch a harvest analytics processor and get the task id in result
+
+            Args:
+                season_duration (int): the duration of the season in days,
+                season_start_day (int): the start day value (1 - 31),
+                season_start_month (int): the start month value (1 - 12),
+                crop (str): the geosys crop code,
+                year (int): the year value,
+                seasonfield_id (sfd) : seasonField geosys uniqueId
+                geometry (str): the geometry to calcultate the analytic (WKT or GeoJSON),
+                harvest_type (Harvest): the type of Harvest analytics (INSEASON/HISTORICAL)
+
+            Returns:
+                taskId (str)
+        """
+
+        if harvest_type == Harvest.IN_SEASON:
+            parameters_profile = "inseason-harvest_default"
```

Comment:
I think, it's good to keep all constants like "inseason-harvest_default" in a separate class within the services repository